tests/test_thinking_budget.py

```
@@ -279,12 +280,12 @@ async def mock_run_cli(cmd, workdir, timeout, env=None, input_text=None):
-        with patch("mcpbr.harnesses._run_cli_command", side_effect=mock_run_cli):
-            with patch("mcpbr.harnesses.shutil.which", return_value="/usr/bin/claude"):
-                try:
-                    await harness.solve(task, "/tmp/test", timeout=1)
-                except Exception:
-                    pass
+        with (
+            patch("mcpbr.harnesses._run_cli_command", side_effect=mock_run_cli),
+            patch("mcpbr.harnesses.shutil.which", return_value="/usr/bin/claude"),
+            contextlib.suppress(Exception),
+        ):
+            await harness.solve(task, "/tmp/test", timeout=1)
```

Ruff (0.15.0)

[error] Probable insecure usage of temporary file or directory: "/tmp/test"

(S108)